src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js

```diff
@@ -37,7 +37,16 @@ define(function () {
                 toolbar: {
                     buttons: {
                         add: {},
-                        deleteSelected: {}
+                        deleteSelected: {},
+                        export: {
+                            options: {
+                                urlParameter: {
+                                    flat: true,
+                                    limit: 999999
```

Comment:
In a discussion with @chirimoya we decided that it is safer to implement that in the `getLimit` method only for the CSV format, because it is less a security issue due to the fact that we are using streams for CSV.